sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/implementation/CosmosEncryptionType.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.encryption.implementation;
+
+/**
+ * Algorithms for use with client-side encryption support in Azure Cosmos DB.
+ */
+public class CosmosEncryptionType {
+    /**
+     * Randomized encryption uses a method that encrypts data in a less predictable manner. Randomized encryption is more secure.
+     */
+    public final static  String RANDOMIZED = "Randomized";
```

Comment:
why String type is preferred over enum?

Reply:
ClientEncryptionIncludedPath includedPath.encryptionType is String , and internally we are comparing these. So having same type will be easier for user